wp-content/themes/core/assets/pcss/login/login.pcss

```diff
@@ -0,0 +1,171 @@
+/* -------------------------------------------------------------------------
+ *
+ * Admin Styles: Login
+ *
+ * ------------------------------------------------------------------------- */
+
+/* -----------------------------------------------------------------------
+ * Login: Body Element
+ * ----------------------------------------------------------------------- */
+
+.login {
+	height: 100%;
+	position: relative;
+	width: 100%;
+	display: table;
+	background: var(--login-background-color);
+}
+
+.login::after {
+	content: "";
+	display: none;
+	width: 50%;
+	height: 100%;
+	position: absolute;
+	top: 0;
+	right: 0;
+	background: var(--login-background-image) center no-repeat;
+	background-size: cover;
+
+	@media (min-width: 960px) {
+		display: block;
+	}
+}
+
+/* -----------------------------------------------------------------------
+ * Login: Button Focus Styling
+ * ----------------------------------------------------------------------- */
+
+.login a,
+.login button,
+.login input[type="submit"] {
+
+	&:focus,
+	&:active {
+		box-shadow: none;
+		outline: 1px dotted;
+	}
+}
+
+/* -----------------------------------------------------------------------
+ * Login: Login Form Wrapper
+ * ----------------------------------------------------------------------- */
+
+.login #login {
+	overflow: hidden;
+	position: relative;
+	z-index: 1;
+	padding: 80px 25px 100px;
+	display: table-cell;
+	width: 100%;
+	vertical-align: middle;
+
+	& > * {
+		max-width: 425px;
+		margin-left: auto;
+		margin-right: auto;
+	}
+
+	@media (min-width: 960px) {
+		width: 50%;
+	}
+}
+
+/* -----------------------------------------------------------------------
+ * Login: Primary Log In Button
+ * ----------------------------------------------------------------------- */
+
+.login.wp-core-ui .button-primary {
+	background: var(--login-primary-color);
+	color: var(--login-white-color);
+	border: 0;
+	box-shadow: none;
+	display: block;
+	width: 100%;
+	padding: 10px;
+	margin: 15px auto 0;
+	font-size: 16px;
+	height: auto;
+	float: none;
+	text-shadow: none;
+
+	&:hover,
+	&:focus,
+	&:active {
+		background: var(--login-secondary-color);
+	}
+}
```

Comment:
Same answer as above, we'd have to separately enqueue these styles to get them to show up on the login screen, as they don't get enqueued otherwise. I think to do this we'd also need some sort of hook to override the button classes which is likely the easier part of this.